templates/download/amd/tab-1.html

```diff
@@ -39,6 +39,17 @@ <h3>Ubuntu Server 22.04 LTS</h3>
 		</div>
 	</div>
 
+	<div class="row p-strip u-no-padding--top">
```

Comment:
since this section is the same for all three tabs, can you please make it a partial and reuse in all three?